src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/quick-start-nrql-condition.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Write your first NRQL alert condition
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+  - Alert conditions
+  - NRQL
+metaDescription: How to start writing NRQL alert conditions with a step by step user case.
+---
+
+Use [NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to create alert [conditions](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-alert-conditions/) with ease.
+
+Your condition only needs two attributes: a [query](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/#specify-data) and a [threshold](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/set-thresholds-alert-condition/).
+
+Follow our step-by-step example to write your first alert condition using the NRQL query builder.
```

Comment:
example --> procedure.

I prefer slightly: "Follow these steps to write..."

Can we assume that this is their first alert condition? :)

I'd follow this sentence up with a description of what they'll have when they're done: "When you're done, you'll have a working alerts condition."